tidb-in-kubernetes/setup.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Deploy TiDB Operator in Kubernetes
+summary: TiDB Operator Deployment
+category: how-to
+---
+
+# Deploy TiDB Operator in Kubernetes
+
+## Prerequisites
+
+Before deploying TiDB Operator, make sure the following items are installed on your machine:
+
+* Kubernetes >= v1.10
+* [DNS addons](https://kubernetes.io/docs/tasks/access-application-cluster/configure-dns-cluster/)
+* [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+* [RBAC](https://kubernetes.io/docs/admin/authorization/rbac) enabled (optional)
+* [Helm](https://helm.sh) version >= v2.8.2 and < v3.0.0
+* Kubernetes v1.12 or higher version is required for zone-aware persistent volumes.
+
+> **Note:** 
+>
+> + Although TiDB Operator can use network volume to persist TiDB data, this process could be very slow due to the redundant replication. It is highly recommended to set up [local volume](https://kubernetes.io/docs/concepts/storage/volumes/#local) for better performance.
+>
+> + Network volumes in a multi-availability zone setup require Kubernetes v1.12 or higher version. It is recommended to use networked volumes to store backup data in `tidb-bakup` chart.
+
+## Deploy Kubernetes cluster
+
+TiDB Operator runs in Kubernetes cluster. You can use one of the methods listed [here](https://kubernetes.io/docs/setup/pick-right-solution/) to set up a Kubernetes cluster. Make sure that the Kubernetes version is v1.10 or higher. If you are using AWS, GKE or local machine, here are quick-start tutorials:
+
+* [Local DinD tutorial](/how-to/get-started/deploy-tidb-from-kubernetes-dind.md)
+* [Google GKE tutorial](/how-to/get-started/deploy-tidb-from-kubernetes-gke.md)
+* [AWS EKS tutorial](/how-to/deploy/orchestrated/tidb-in-kubernetes/aws-eks.md)
+
+If you are deploying in a different environment, a proper DNS addon must be installed in the Kubernetes cluster. You can follow the [official documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-dns-cluster/) to set up a DNS addon.
+
+TiDB Operator uses [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) to persist the data of TiDB cluster (including the database, monitoring data, backup data), so Kubernetes cluster must provide at least one kind of persistent volume. For better performance, it is recommended to use local SSD disk as the volumes. Follow [this step](#local-persistent-volume) to auto-provision local persistent volumes.
+
+It is suggested to enable [RBAC](https://kubernetes.io/docs/admin/authorization/rbac) in the Kubernetes cluster. Otherwise, you need to set `rbac.create` to `false` in the `values.yaml` of both `tidb-operator` and `tidb-cluster` charts.
+
+Because TiDB by default will use many file descriptors, the [worker node](https://access.redhat.com/solutions/61334) and its Docker daemon's `ulimit` must be configured to `1048576` or bigger:
```

Comment:
```suggestion
Because TiDB uses many file descriptors by default, the [worker node](https://access.redhat.com/solutions/61334) and its Docker daemon's `ulimit` must be configured to `1048576` or bigger:
```